### PR TITLE
replace DOTNET_PRINT_TELEMETRY_MESSAGE with DOTNET_NOLOGO  /7

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/host-and-deploy/linux-nginx.md](https://github.com/dotnet/AspNetCore.Docs/blob/fa2253217e7e9b2e058cdfa1b8e087bc1b299f4d/aspnetcore/host-and-deploy/linux-nginx.md) | [Host ASP.NET Core on Linux with Nginx](https://review.learn.microsoft.com/en-us/aspnet/core/host-and-deploy/linux-nginx?branch=pr-en-us-31399) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@am11 @wli3 please review

see https://github.com/dotnet/sdk/issues/9734

File: aspnetcore/host-and-deploy/linux-nginx/includes/linux-nginx5.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ KillSignal=SIGINT
 SyslogIdentifier=dotnet-example
 User=www-data
 Environment=ASPNETCORE_ENVIRONMENT=Production
-Environment=DOTNET_PRINT_TELEMETRY_MESSAGE=false
+Environment=DOTNET_NOLOGO=false
```
Comment (am11) [inline aspnetcore/host-and-deploy/linux-nginx/includes/linux-nginx5.md:202]: There is one more entry in this document.

**Comment by Bykiev** [comment]
@Rick-Anderson, sorry, maybe `DOTNET_NOLOGO` variable should be set to `true` in ASP NET Core app?

File: aspnetcore/host-and-deploy/linux-nginx.md [hunk 2]
```diff
@@ -264,7 +264,7 @@ KillSignal=SIGINT
 SyslogIdentifier=dotnet-example
 User=www-data
 Environment=ASPNETCORE_ENVIRONMENT=Production
-Environment=DOTNET_PRINT_TELEMETRY_MESSAGE=false
+Environment=DOTNET_NOLOGO=false
```
Comment (Bykiev) [inline aspnetcore/host-and-deploy/linux-nginx.md:267]: `DOTNET_NOLOGO` variable should be set to `true`

File: aspnetcore/host-and-deploy/linux-nginx/includes/linux-nginx5.md (hunk 1 above)
Comment (Bykiev) [inline aspnetcore/host-and-deploy/linux-nginx/includes/linux-nginx5.md:202]: `DOTNET_NOLOGO` variable should be set to `true`

**Pull request merged**